_posts/camazotz/2016-02-02-NavaneetsCh5Exercises.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+author: camazotz
+title: "Navaneet's Chapter 5 Exercises!"
+---
+
+Chapter 5 Exercise 1:
+
+<iframe src="https://trinket.io/embed/python/34792a7e4b" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+Chapter 5 Exercise 2:
+
+<iframe src="https://trinket.io/embed/python/56c04561c9" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
```

Comment:
Clever idea with the inf but there's an error in your logic elsewhere.  For inputs of 0, 5, 10 I got a max of 5 and a min of `inf`.